src/index.ts

```diff
@@ -361,7 +375,7 @@ export class GossipSub extends EventEmitter<GossipsubEvents> implements Initiali
     cancel: () => void
   } | null = null
 
-  constructor(options: Partial<GossipsubOpts> = {}) {
+  constructor(components: GossipSubComponents, options: Partial<GossipsubOpts> = {}) {
```

Comment:
This is a breaking change?

Reply:
Yeah, as noted in the PR title & description